scripting/get5/pausing.sp

```diff
@@ -59,13 +77,24 @@ public Action Command_Pause(int client, int args) {
 
   // If the pause will need explicit resuming, we will create a timer to poll the pause status.
   bool need_resume = Pause(g_FixedPauseTimeCvar.IntValue, MatchTeamToCSTeam(team));
+  EventLogger_PauseCommand(team, PauseType_Tac);
+  LogDebug("Calling Get5_OnMatchPaused(team=%d, pauseReason=%d)", team, PauseType_Tac);
+  Call_StartForward(g_OnMatchPaused);
+  Call_PushCell(team);
+  Call_PushCell(PauseType_Tac);
+  Call_Finish();
   if (IsPlayer(client)) {
     Get5_MessageToAll("%t", "MatchPausedByTeamMessage", client);
   }
 
   if (IsPlayerTeam(team)) {
     if (need_resume) {
+      g_PauseTimeUsed = g_PauseTimeUsed + g_FixedPauseTimeCvar.IntValue - 1;
       CreateTimer(1.0, Timer_PauseTimeCheck, team, TIMER_REPEAT | TIMER_FLAG_NO_MAPCHANGE);
+      // Keep track of timer, since we don't want several timers created for one pause checking instance.
```

Comment:
Are you sure you need to keep track of it? This would be the only case of that in the project.

Reply:
Hey! Thanks for taking the time review, it's much appreciated!

I think initially I wanted to keep track of this here because I had an initial assumption/testing that a timer would be created each time a person called the `!pause` command, which would create a new timer and erroneously cause another timer, and thus event firing, to be sent off when the timer finished. However, I don't think this is the case upon further review. I'll make that change and remove the handler and test that out here :) Thanks again!